src/notation/internal/notationinteraction.cpp

```diff
@@ -752,43 +771,32 @@ void NotationInteraction::drag(const PointF& fromPos, const PointF& toPos, DragM
 
     score()->update();
 
-    std::vector<LineF> anchorLines;
-    for (const EngravingItem* e : m_dragData.elements) {
-        QVector<LineF> elAnchorLines = e->dragAnchorLines();
-        const Ms::EngravingItem* page = e->findAncestor(ElementType::PAGE);
-        const PointF pageOffset((page ? page : e)->pos());
-
-        if (!elAnchorLines.isEmpty()) {
-            for (LineF& l : elAnchorLines) {
-                l.translate(pageOffset);
-                anchorLines.push_back(l);
+    if (isGripEditStarted()) {
+        updateAnchorLines();
```

Comment:
this was definitely wrong, was causing anchor lines to disappear once you started dragging grip handles